content/en/news/releases/1.20.x/announcing-1.20/_index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Announcing Istio 1.20.0
+linktitle: 1.20.0
+subtitle: Major Release
+description: Istio 1.20 Release Announcement.
+publishdate: 2023-11-14
+release: 1.20.0
+aliases:
+- /news/announcing-1.20
+- /news/announcing-1.20.0
+---
+
+We are pleased to announce the release of Istio 1.20. This is the last Istio release of 2023. We would like to thank the
```

Comment:
Is there a document with this file's content?  This is basically the marketing blog for the launch, and I'm happy to "punch these up" and make it more readable.

Reply:
Thanks! I'm going to create a one and paste the link here.

Reply:
@craigbox Here's the doc link: https://docs.google.com/document/d/1EO2cbtKYKH1eC4FS3o2l9pKANA308JdEbwbH6TTLblA/edit?usp=sharing